src/main/java/uk/gov/gchq/palisade/service/request/ResourceDetails.java

```diff
@@ -41,6 +42,21 @@ public ResourceDetails(final String fileName, final String type, final String fo
         this.format = format;
```

Comment:
use setters

Reply:
Ok, added the setters.

Reply:
lines 37, 38, 39 (null checks) arent needed, and 40, 41, 42 should use setters